### PR TITLE
use production environment in staging

### PR DESCRIPTION
**Pull request by tkellen**

### CODE REVIEW

**Comment by tkellen** [comment]
ready for review

File: package.json [hunk 1]
```diff
@@ -14,7 +14,6 @@
   "scripts": {
     "build": "webpack",
     "build:development": "NODE_ENV=development webpack",
-    "build:staging": "NODE_ENV=staging webpack",
```
Comment (gnarf) [inline package.json]: Hrm... 

I can see us wanting to be able to do a no_mangle as an uglify option in staging some day...

Comment (tkellen) [inline package.json]: Wait until that day arrives?

Comment (gnarf) [inline package.json]: fair enough - was wondering if that day way here yet, but its easy enough to undo this then :)

**Pull request merged**